config/create-common.toml

```diff
@@ -12,14 +12,14 @@
 		[worldgen.v2.striated_ores_nether]
 			#.
 			#Range: > 0
-			clusterSize = 32
+			clusterSize = 0

```

Comment:
make sure this isn't causing an error, since it expects a value greater than 0 even though it's disabled.

Reply:
Appears to be fine ^^ But for safety (as I saw config swapper errors) I will change this back to normal